spec/mongoid/collection_configurable_spec.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+class CollectionConfigurableCapped
```

Comment:
I think these should be defined in a `before(:all)` block, and call `remove_const` on them in an `after(:all)`